tests/testcases/test_user.py

```diff
@@ -55,10 +55,6 @@ def test_get_user(self):
             os.environ['PKEXEC_UID'] = '1000'
             self.assertNotEqual(get_user(), 'root')
 
-    def test_get_home(self):
```

Comment:
`def test_get_home(self):`

this line was actually allright. Since you are testing get_home below.

Reply:
anyway, I don't care that much. If this is ready I'll hit merge, just let me know